overrides/grid/column/Action.js

```diff
@@ -1,6 +1,18 @@
 Ext.define('Jarvus.ext.glyphs.override.grid.column.Action', {
     override: 'Ext.grid.column.Action',
 
+    beforeRender: function() {
+        var me = this,
+            itemsCount = me.items.length;
+
+        // auto-size the column
+        me.setWidth(
+            itemsCount * 16 // icons
+            + (itemsCount - 1) * 4 // space between
+            + 10 // cell padding
+        );
```

Comment:
Like all of us, template method overrides should call their parents:

```suggestion
        );

        me.callParent();
```